cora/scripts/makesky.py

```diff
@@ -12,9 +12,9 @@
 @click.option('--channels', help='Select a range of frequency channels',
               type=(int, int), metavar='CSTART CSTOP', default=(None, None))
 @click.option('--channel-bin', help='If set, average over BIN channels', metavar='BIN', type=int, default=1)
-@click.option('--freq-mode', type=click.Choice(['centre', 'edge']), default='centre',
-              help=('Choose whether FSTART and FSTOP are the very edges of the band, '+
-                    'or whether they are the centre frequencies (default: centre).'))
+@click.option('--freq-mode', type=click.Choice(['centre', 'chime', 'edge']), multiple=True, default='centre',
+              help=('Choose whether FSTART and FSTOP are the centre frequencies, the frequencies for ' +
+                    'the CHIME telescope or the very edges of the band, (default: centre).'))
```

Comment:
It's not clear why we need this option. Can't you get the same thing by just setting `--channels 0 1024`?

Reply:
True. I missed that and reverted that change.